### PR TITLE
feat(sms.batches): add pending page

### PR DESCRIPTION
**Pull request by JeremyDec**
ref: MANAGER-5201
Signed-off-by: Jérémy De-Cesare <jeremy.de-cesare@corp.ovh.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/sms-campaign-statistics`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5201
| License          | BSD 3-Clause

## Description

Add pending batch list page

### CODE REVIEW

File: packages/manager/modules/sms/src/sms/batches/pending/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import '@uirouter/angularjs';
+
+import cancel from './cancel';
+import component from './telecom-sms-batches-pending.component';
+import routing from './routing';
+
+const moduleName = 'ovhManagerSmsBatchesPendingModule';
+
+angular
+  .module(moduleName, ['ui.router', cancel])
```
Comment (radireddy) [inline packages/manager/modules/sms/src/sms/batches/pending/index.js:11]: why not lazyload the module?

File: packages/manager/modules/sms/src/sms/batches/pending/routing.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        }),
+      goToDetails: /* @ngInject */ ($state) => (batch) =>
+        $state.go('sms.service.batches.details', {
+          batchId: batch.id,
+          previousState: 'sms.service.batches.pending',
+        }),
+      plannedBatches: /* @ngInject */ (batches, batchStatuses) =>
+        batches.filter(({ status }) =>
+          [
+            batchStatuses.PENDING,
```
Comment (radireddy) [inline packages/manager/modules/sms/src/sms/batches/pending/routing.js:24]: 👍

File: packages/manager/modules/sms/src/sms/batches/pending/telecom-sms-batches-pending.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<header>
+    <oui-back-button data-on-click="$ctrl.goBack()">
+        <span data-translate="sms_batches_pending"></span>
+    </oui-back-button>
+</header>
+
+<section class="container-fluid">
+    <oui-message
+        type="error"
```
Comment (radireddy) [inline packages/manager/modules/sms/src/sms/batches/pending/telecom-sms-batches-pending.html:9]:
```suggestion
        data-type="error"
```

File: packages/manager/modules/sms/src/sms/batches/pending/telecom-sms-batches-pending.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        data-ng-if="$ctrl.error"
+        dismissable
```
Comment (radireddy) [inline packages/manager/modules/sms/src/sms/batches/pending/telecom-sms-batches-pending.html:11]:
```suggestion
        data-dismissable
```

File: packages/manager/modules/sms/src/sms/batches/pending/telecom-sms-batches-pending.html [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        on-dismiss="$ctrl.error = null"
+    >
+        <span
+            data-translate="sms_batches_error"
+            data-translate-values="{ 'error': $ctrl.error }"
+        ></span>
+    </oui-message>
+
+    <oui-datagrid
+        page-size="5"
```
Comment (radireddy) [inline packages/manager/modules/sms/src/sms/batches/pending/telecom-sms-batches-pending.html:21]: a lot of missing data-

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/sms/src/sms/batches/pending/telecom-sms-batches-pending.html (hunk 5 above)
Comment (marie-j) [reply packages/manager/modules/sms/src/sms/batches/pending/telecom-sms-batches-pending.html:21]: @radireddy As mentionned in several pull requests, `data-` are no longer requested on custom components like ui-kit one. If you think we should keep them, maybe we can discuss about it :)

**Review by marie-j — COMMENTED** [review]
Changes are looking good to me but commit history could be improved. I don't think 5 commits are necessary to add a missing deps to both module and app, 1 fix commit such as `fix(deps): add missing dependency` looks sufficient

**Review by marie-j — APPROVED** [review]

**Pull request merged**